R/available_forecasts.R

```diff
@@ -37,10 +37,9 @@ available_forecasts <- function(data,
                                 by = NULL,
                                 collapse = c("quantile", "sample")) {
 
-  check_data <- check_forecasts(data)
-
-  data <- check_data$cleaned_data
-  forecast_unit <- check_data$forecast_unit
+  data <- validate(data)
```

Comment:
Is this a unique enough name? Is it really good practice to go with something so generic?

Reply:
I would have thought `validate_forecasts` would be the way to go?

Reply:
Created a new issue and will do in a new PR that is nice and clean and modular and separate from other PRs :) https://github.com/epiforecasts/scoringutils/issues/376